### PR TITLE
Re-add tools guide

### PR DESCRIPTION
**Pull request by DannyS03**
showcase tools guide with updated references

### CODE REVIEW

File: docs/src/guide/develop/quick-start/which-tools.md [hunk 1]
```diff
@@ -1,62 +1,32 @@
...
 #### palomad
 
-The command line interface and node daemon for interacting with the Paloma blockchain. Use `palomad` to run a full node or interact with the chain. Follow the [`palomad` install guide](palomad/install-palomad.md) to get started.
+The command line interface and node daemon for interacting with the Paloma blockchain. Use `palomad` to run a full node or interact with the chain. Follow the [`palomad` install guide](../palomad/install-palomad) to get started.
 
-#### Finder
+#### Explorer
 
-Paloma’s multi-purpose [block explorer](https://finder.Paloma.money/).
+Paloma’s multi-purpose [block explorer](https://paloma.explorers.guru/).
```
Comment (verabehr) [inline docs/src/guide/develop/quick-start/which-tools.md:29]: please add that this is a community project

File: docs/src/guide/develop/quick-start/which-tools.md [hunk 2]
```diff
@@ -1,62 +1,32 @@
...
-
-Use paloma.js to create bots, mint NFTs, and build out backend services. Follow the [paloma.js tutorial](Paloma-js/getting-started.md) to get started.
-
-Use paloma.py for on-chain analysis, Jupyter Notebooks, data processing, and other analytical use cases suited for Python. Follow the [paloma.py tutorial](https://paloma.github.io/paloma.py/tutorial.html) to get started.
-
-## Wallet Provider and Templates
-
-Wallet Provider makes it easy to connect a React-based web app to Paloma Station. There are multiple [templates](https://templates.Paloma.money/) available for frontend frameworks like Vue, React, and Vite. Follow the [Wallet Provider tutorial](wallet-provider/wallet-provider-tutorial.md) to get started.
+Wallet provider makes it easy to connect a React-based web app to Paloma Station. 
+Follow the [Wallet Provider tutorial](../wallet-provider/wallet-provider-tutorial.md) to get started.
```
Comment (verabehr) [inline docs/src/guide/develop/quick-start/which-tools.md:16]: please remove paloma.py and wallet for now

**Review by verabehr — CHANGES_REQUESTED** [review]
small change request

File: docs/src/guide/develop/quick-start/which-tools.md (hunk 1 above)
Comment (DannyS03) [reply docs/src/guide/develop/quick-start/which-tools.md:29]: addressed

File: docs/src/guide/develop/quick-start/which-tools.md (hunk 2 above)
Comment (DannyS03) [reply docs/src/guide/develop/quick-start/which-tools.md:16]: addressed

**Pull request merged**